integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonCreateTableCommand.scala

```diff
@@ -122,7 +122,6 @@ case class CarbonCreateTableCommand(
           val tablePath = tableIdentifier.getTablePath
           val carbonRelation = CarbonSparkUtil.createCarbonRelation(tableInfo, tablePath)
           val rawSchema = CarbonSparkUtil.getRawSchema(carbonRelation)
-          sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
```

Comment:
Many places in the code we have, 
sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null) --> please check and handle if required in all the places.

Reply:
@ajantha-bhat Handled the comment..please review